src/coverlet.collector/coverlet.collector.csproj

```diff
@@ -61,12 +63,11 @@
   <ItemGroup>
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\coverlet.core\coverlet.core.csproj" PrivateAssets="All" />
   </ItemGroup>
-  
-  <Target Name="SetNuspecPackageVersion" DependsOnTargets="GetBuildVersion" BeforeTargets="GenerateNuspec">
-    <PropertyGroup>
-      <NuspecProperties>$(NuspecProperties);version=$(NuGetPackageVersion);configuration=$(Configuration)
-    </NuspecProperties>
-  </PropertyGroup>
-  </Target>
 
+  <Target Name="PackBuildOutputs">
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="$(TargetDir)\*.dll" PackagePath="build\netstandard1.0\" />
```

Comment:
This was my missing piece :)

Reply:
@vagisha-nidhi do you have concerns about `netstandard1.0`?

Reply:
The nupkg will contain all dlls and targets in `netstandard1.0` folder right? That shouldn't be a problem.

Reply:
yup